sdks/python/src/opik/console_utils.py

```diff
@@ -0,0 +1,62 @@
+from typing import Callable
+import logging
+import tqdm
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def _in_jupyter_environment() -> bool:
```

Comment:
We have those functions in `opik.environment`, I think you can just add `get_tqdm_for_current_environment` function to this module and remove the one you've added

Reply:
Oops... didn't see them. And they are also here: sdks/python/src/opik/configurator/interactive_helpers.py